### DISCUSSION TITLE
The engine starts, passes in the board data, returns the result of the move is not empty

### DISCUSSION THREAD

**Discussion by jialaolian11** [fairy-stockfish/Fairy-Stockfish#694] [Q&A]
Hello, Sir, I looked at my cpu and memory capacity, they have a lot of space, but my program passed in the chessboard data and access the engine, the engine returned the result of the move is empty, please tell me how to troubleshoot this problem

**Comment by ianfab** [comment]
Please describe the exact steps (ideally the UCI input you are providing), then I can try to help.

**Comment by jialaolian11** [comment]
uci
setoption name Skill Level value 5
position fen rnbkqbnr/pppppppp/8/8/8/8/PPPPPPPP/RNBKQBNR w KQkq - 0 1 
go depth  1

**Reply by jialaolian11** [reply]
This phenomenon is not necessary, in the occasional case will be the beginning of the game will be a continuous move for empty, or in the course of the move for empty

**Comment by jialaolian11** [comment]
Could you please tell how to troubleshoot this problem
